### PR TITLE
change versino for npm publish

### PR DESCRIPTION
**Pull request by diced**
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus pellentesque ullamcorper enim non fringilla. Nam at dui commodo, cursus ex sed, vehicula dui. Pellentesque id ex pulvinar, fringilla erat ac, varius ex. Aliquam augue eros, egestas non ultrices eget, dapibus sed augue. Integer suscipit ac elit a consectetur. Ut auctor lorem eget libero vehicula, in feugiat erat iaculis. Integer ante diam, euismod sit amet vulputate molestie, finibus sit amet eros. Nulla ornare enim dapibus porta finibus. Aliquam feugiat eu lectus a eleifend. Vivamus tincidunt semper velit, luctus molestie tellus malesuada volutpat.

Nunc a ullamcorper turpis. Phasellus fringilla magna et scelerisque pretium. Curabitur finibus tortor sem, eu euismod risus pulvinar quis. Proin sagittis blandit leo, id dapibus leo rhoncus fermentum. Ut consectetur consequat erat. Praesent eros quam, dictum eu nulla mollis, aliquet rhoncus tortor. Etiam sodales vulputate justo vitae iaculis. Mauris condimentum aliquam est nec convallis. Praesent venenatis interdum nisl, ac pretium dolor pretium nec. Aliquam erat volutpat.

Phasellus auctor volutpat commodo. Pellentesque molestie imperdiet purus, eu blandit metus tempor eget. Ut suscipit ligula non arcu sollicitudin, sollicitudin iaculis justo congue. Maecenas finibus semper elit eu fringilla. Praesent suscipit porttitor erat ac sagittis. Aenean pulvinar odio quam, sit amet maximus dolor tempus id. Quisque mollis consequat nibh non fringilla. Quisque interdum tempor metus congue bibendum. Fusce euismod massa nec ante blandit, sed finibus odio sodales. Maecenas vitae augue in sapien accumsan bibendum. Integer purus ex, ullamcorper non mollis id, aliquam ac ipsum. Ut eu gravida orci. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris imperdiet congue diam, lobortis aliquam ligula ornare vitae. Fusce odio dolor, faucibus eget metus ac, luctus ultrices nibh.

Nulla facilisi. Integer et pellentesque quam, at tincidunt nunc. Etiam placerat, turpis id porta bibendum, risus nibh pretium tortor, ut pulvinar tortor nunc sed eros. Morbi porttitor a odio accumsan lobortis. Nullam pharetra porta interdum. Aliquam tempor euismod turpis, nec semper metus consequat at. Vestibulum eget turpis risus. Ut eu lectus enim. Aliquam tincidunt velit libero, id ornare enim porta a. Donec dictum lectus nec leo consectetur vestibulum.

Etiam quis mauris aliquam, posuere velit eu, aliquam sapien. Vivamus semper, nisi at commodo commodo, nisi neque aliquet dui, eget malesuada dolor sapien et erat. Etiam luctus nisl cursus libero ullamcorper tristique. Fusce vestibulum risus ut imperdiet euismod. Nulla facilisi. Etiam nec lacus aliquet, pretium velit sed, aliquet odio. Phasellus imperdiet sollicitudin erat, eu semper arcu gravida aliquet. Phasellus feugiat sed nibh vitae iaculis.

### CODE REVIEW

**Comment by diced** [comment]
pog comit

**Review by advilm — APPROVED** [review]

**Comment by diced** [comment]
pog comit

File: README.md [hunk 1]
```diff
@@ -19,11 +19,11 @@
 
 ### Installing via NPM.
 
-```$ npm i ytsearcher@1.2.3```
+```$ npm i ytsearcher@1.2.4```
```
Comment (diced) [inline README.md]: use latest

**Review by wzhouwzhou — APPROVED** [review]

**Review by advilm — APPROVED** [review]
M

File: README.md [hunk 2]
```diff
@@ -19,11 +19,11 @@
 
 ### Installing via NPM.
 
-```$ npm i ytsearcher@1.2.3```
+```$ npm i ytsearcher@latest```
 
 ### Why ytsearcher?
 
-• Modern and trustworthy (promises, es6, up-to-date) with thousands of downloads.
+• Modern and trustworthy (typings(!), promises, es6, up-to-date) with thousands of downloads.
```
Comment (tbnritzdoge) [inline README.md:26]: Not really using es6 features but shrug

**Review by diced — COMMENTED** [review]
> M

File: README.md (hunk 2 above)
Comment (diced) [reply README.md:26]: no one asked

**Comment by diced** [comment]
Duplicate of #69420

**Review by wzhouwzhou — APPROVED** [review]
Looks great now, thanks!

**Review by TacticalTechJay — APPROVED** [review]
Although these are petty changes, I approve.

Oh yeah, nice gamer code bois.

**Review by rpxs — APPROVED** [review]
greatest commit i have witnessed in my relatively small lifetime

**Review by wzhouwzhou — APPROVED** [review]

**Comment by advilm** [comment]
@diced nice job :)

**Review by wzhouwzhou — APPROVED** [review]

**Review by advilm — APPROVED** [review]
:))))

**Comment by diced** [comment]
> @diced nice job :)

NO

**Comment by rpxs** [comment]
what

**Comment by diced** [comment]
aWSZOYUWZHOU REVIEW PR AAA

**Review by wzhouwzhou — APPROVED** [review]
Re-opened version looks even better now.

**Comment by diced** [comment]
thats the spirit

**Comment by advilm** [comment]
L9 version bump technology :fire::rocket:

**Review by wzhouwzhou — APPROVED** [review]
Approving now with the proper label

**Comment by diced** [comment]
its amazing what amount of work goes behind big companies when pushing new releases

**Review by rpxs — APPROVED** [review]
I approve of these changes due to the fact that they are absolutely necessary. I see the commit as an enhancement to the infrastructure of this package, and I think that merging this pull request will create great change in the poggers ytsearcher community.

**Review by Excigma — APPROVED** [review]

**Review by Excigma — APPROVED** [review]

**Review by rpxs — APPROVED** [review]
eating taco bell tmrw

**Comment by advilm** [comment]
@Vetlix review these changes

**Comment by diced** [comment]
@Vetlix please review

**Comment by advilm** [comment]
@Vetlix :rage:

**Review by rpxs — APPROVED** [review]

**Review by wzhouwzhou — APPROVED** [review]
Excellent PR description

**Review by TacticalTechJay — APPROVED** [review]

**Review by TacticalTechJay — CHANGES_REQUESTED** [review]
jk lol, change this code.

File: README.md (hunk 2 above)
Comment (TacticalTechJay) [reply README.md:26]: @tbnritzdoge Do you have any complaints about this change? :D

**Review by rpxs — APPROVED** [review]
tostadas on the menu

**Comment by rpxs** [comment]
WAIT WRONG SITE SORRY SORRY

**Comment by TacticalTechJay** [comment]
> WAIT WRONG SITE SORRY SORRY

Hol up

File: README.md (hunk 2 above)
Comment (wzhouwzhou) [reply README.md:26]: Doesn't seem to be any truth behind the comment. Thank goodness for OSS.

**Review by wzhouwzhou — APPROVED** [review]

**Review by rpxs — APPROVED** [review]

**Comment by rpxs** [comment]
@Vetlix review these commits or this PR will not be merged :trollface:

**Comment by diced** [comment]
@wzhouwzhou

**Review by Vetlix — APPROVED** [review]

**Comment by diced** [comment]
Ok this pr has enough reviewers ready to merge

**Comment by rpxs** [comment]
agreed

**Comment by diced** [comment]
ok

**Pull request merged**